gravitee-apim-console-webui/src/management/api/api-traffic-v4/webhook-logs/components/webhook-logs-more-filters/components/webhook-logs-more-filters-form/webhook-logs-more-filters-form.component.html

```diff
@@ -0,0 +1,66 @@
+<!--
+
+    Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+            http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+@if (datesForm) {
+  <form [formGroup]="datesForm">
+    <div class="more-filters__body__block">
+      <span class="more-filters__body__title">Date</span>
+      <mat-form-field>
+        <mat-label>Period</mat-label>
+        <mat-select formControlName="period" aria-label="Period selection">
+          @for (period of periods; track period.value) {
+            <mat-option [value]="period">{{ period.label }}</mat-option>
+          }
+        </mat-select>
+      </mat-form-field>
+
+      <mat-form-field>
+        <mat-label>From</mat-label>
+        <input matInput [owlDateTime]="fromDateTimePicker" formControlName="from" />
+        <mat-icon [owlDateTimeTrigger]="fromDateTimePicker" matSuffix svgIcon="gio:nav-arrow-down"></mat-icon>
+        <owl-date-time #fromDateTimePicker></owl-date-time>
+      </mat-form-field>
+
+      <mat-form-field>
+        <mat-label>To</mat-label>
+        <input matInput [owlDateTime]="toDateTimePicker" [min]="minDate" formControlName="to" />
+        <mat-icon [owlDateTimeTrigger]="toDateTimePicker" matSuffix svgIcon="gio:nav-arrow-down"></mat-icon>
+        <owl-date-time #toDateTimePicker></owl-date-time>
+        @if (datesForm.get('to').hasError('owlDateTimeMin')) {
```

Comment:
Typed forms can help with assessment and accessing data.
```suggestion
        @if (datesForm.controls.to.hasError('owlDateTimeMin')) {
```

Reply:
Thank you for this. Addressed in [282bfa9](https://github.com/gravitee-io/gravitee-api-management/pull/14023/commits/282bfa968140bbbf2c5c27a08ac301ba9ad25b30).